src/inspector/view/qml/MuseScore/Inspector/notation/notes/HeadSettings.qml

```diff
@@ -11,200 +11,206 @@ FocusableItem {
 
     property QtObject model: null
 
-    implicitHeight: contentColumn.height
+    implicitHeight: noteheadGridView.height
     width: parent.width
 
-    Column {
-        id: contentColumn
+    ScrollView {
+        height: noteheadGridView.height
+        width: parent.width + 10
+        clip: true
 
-        width: parent.width
+        Column {
+            id: contentColumn
+            width: root.width - (2 * anchors.leftMargin)
+            spacing: 16
+            anchors.left: parent.left
+            anchors.leftMargin: 5
```

Comment:
we use values that are multiples of 4 (4, 8, 12, 16 ...) in the margins, spacings, width, etc.